content/en/platform/corda/4.8/open-source/node-docker-deployments.md

```diff
@@ -0,0 +1,193 @@
+---
+aliases:
+- /head/node-docker-deployments.html
+- /HEAD/node-docker-deployment.html
+- /node-docker-deployments.html
+date: '2020-04-07T12:00:00Z'
+menu:
+  corda-os-4-8:
+    identifier: corda-os-4-8-node-docker-deployments
+    parent: corda-os-4-8-corda-nodes-index
+    weight: 1150
+tags:
+- node
+- docker
+title: Docker deployments
+---
+
+# Docker deployments
+
+This page contains a repository of example manual node configuration operations using Docker. Before executing any of these commands,
+ensure the [Corda Docker image](docker-image.md) has been correctly configured.
```

Comment:
We don't talk about configuring a docker image.  We just want to make sure we use the correct docker command or docker-compose command.

Reply:
I think removing 'configuration' clears things up a little in that case